js/adapt-contrib-gmcq.js

```diff
@@ -49,6 +49,51 @@ define([
 
       this.$el.toggleClass('has-column-layout', isLarge);
       this.$('.js-mcq-item').css('width', isLarge ? (100 / columns) + '%' : '');
+    },
+
+    updateMarking: function () {
+
+      var isInteractive = this.model.isInteractive();
+      var canShowMarking = this.model.get('_canShowMarking');
+      var ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
+
+      this.model.getChildren().forEach(function(itemModel) {
+
+        var index = itemModel.get('_index');
+        var $itemInput = this.$('.js-item-input').filter('[data-adapt-index="' + index + '"]');
+        var $item = $itemInput.parents('.js-mcq-item');
+
+        if (isInteractive || !canShowMarking) {
+          // Remove item marking
+          $item.removeClass('is-correct is-incorrect');
+          $itemInput.attr('aria-label', [
+            $.a11y_normalize(itemModel.get('text')),
+            '. ',
+            $.a11y_normalize(itemModel.get('_graphic').alt)
```

Comment:
There is a new API for this:
```suggestion
            Adapt.a11y.normalize(itemModel.get('text')),
            '. ',
            Adapt.a11y.normalize(itemModel.get('_graphic').alt)
```
See also lines 91 & 93.

@oliverfoster, probably best to leave the switch to the direct `a11y.normalize()` for the React PR.

Reply:
Nah, do it now.

Reply:
> See also lines 91 & 93.

Future note: this bit wasn't done.

Reply:
Apologies, my mistake.